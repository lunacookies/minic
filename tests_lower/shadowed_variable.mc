func a {
	x := 10;
	x := 20;
}
