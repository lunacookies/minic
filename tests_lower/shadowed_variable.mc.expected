func a
	var x i64;
	{
		set x = 10;
		<missing>
	}
tests_lower:20..28: error: cannot shadow existing variable
tests_lower:10..18: warning: unused variable
